bus-mapping/src/circuit_input_builder/execution.rs

```diff
@@ -360,10 +390,8 @@ pub struct CopyEvent {
     pub log_id: Option<u64>,
     /// Value of rw counter at start of this copy event
     pub rw_counter_start: RWCounter,
-    /// Represents the list of (bytes, is_code, mask) copied during this copy event
-    pub bytes: CopyEventSteps,
-    /// Represents the list of (bytes, is_code, mask) read to copy during this copy event
-    pub aux_bytes: Option<CopyEventSteps>,
+    /// Represents the list of bytes related during this copy event
+    pub copy_bytes: CopyBytes,
```

Comment:
Here we could use the structure `CopyStep` instead of `(u8, bool, bool)`

Reply:
`prev_byte` doesn't care about `is_code`, `is_mask` . use `CopyStep` may import more unused fields.

Reply:
I meant to add only value_prev to the existing tuple, next to value. is_mask and is_code are part of a step, and value and value_prev also part of the same step.